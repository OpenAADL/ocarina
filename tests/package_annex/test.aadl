package Test
public

system A
end A;

system implementation A.impl
end A.impl;

annex real_specification {**
	theorem test_theorem
	foreach s in system_set do
		check(1=1);
	end test_theorem;
**};

end Test;