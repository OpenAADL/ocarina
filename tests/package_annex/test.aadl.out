







package Test
public
  system A
  end A;

  system implementation A.impl
  end A.impl;

    annex real_specification {**

--  test_theorem

theorem test_theorem
foreach s in system_set do

check ((1 = 1));
end test_theorem;


    **};
end Test;

